src/wpanctl/tool-cmd-commissioner.c

```diff
@@ -323,6 +325,11 @@ int tool_cmd_commissioner(int argc, char* argv[])
 				goto bail;
 			}
 
+			// XXX: interpret PSK string as binary PSKd
+			memcpy(pskd, psk, psk_len);
```

Comment:
Possible memory corruption...`pskd` is defined as `uint8_t pskd[20];` (20 bytes long)... what is psk_len is larger than 20 bytes... ? 
We check for it to be less than `PSK_MAX_LENGTH` which is 32...